README.md

```diff
@@ -113,7 +113,7 @@ The simulators we provide are all open-source and have their own dedicated docum
 - [XBeach](https://oss.deltares.nl/web/xbeach/)
 - [GROMACS](https://www.gromacs.org/)
 
-To learn how to use these simulators with Inductiva API, check the example below and the [Simulators section](https://github.com/inductiva/inductiva/tree/main/inductiva/simulators/README.md)
+To learn how to use these simulators with Inductiva API, check the example below and the [Simulators section](https://github.com/inductiva/inductiva/tree/main/inductiva/simulators#README.md).
```

Comment:
Assuming you checked it points to the right thing, LGTM

Reply:
Just did.

Reply:
I think you wanted this

https://github.com/inductiva/inductiva/tree/main/inductiva/simulators#simulators

Reply:
That is what I did, after you have already commented :)